redshift.tf

```diff
@@ -6,5 +6,6 @@ resource "aws_redshift_cluster" "redshiftEncryptedFalse" {
   node_type          = "dc1.large"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Use KMS CMK customer-managed keys for Redshift clusters**
 
**Vulnerability**
Use customer-managed KMS keys instead of AWS-managed keys, to have granular control over encrypting and encrypting data. Encrypt Redshift clusters with a Customer-managed KMS key. This is a recommended best practice.
 
**Remediation**
In the AWS Redshift console, configure the Redshift cluster to use an encrypted database, using the AWS KMS, according to this procedure:https://docs.aws.amazon.com/redshift/latest/mgmt/managing-clusters-console.html#create-cluster . Notes: 1. You must enable encryption when you launch the Redshift cluster, before data is placed on it. 2. Encryption on a cluster is immutable, and cannot be reversed.